pyClient/test_commands/scenario.py

```diff
@@ -409,16 +407,51 @@ def charlie_corrupt_bob_deposit(
     assert(result_corrupt2 is None), \
         "Charlie managed to corrupt Bob's deposit the second time!"
 
+    # Case3: Charlie uses the correct mix data, but attempts to send the mix
+    # call from his own address (thereby receiving the output).
+    result_corrupt3 = None
+    try:
+        joinsplit_sig_bob = joinsplit.joinsplit_sign(
+            joinsplit_keypair,
+            bob_eth_address,
```

Comment:
Shouldn't the address here be changed to Charlie's?

Reply:
Here we are testing the case where bob has created the mix parameters (and included his own eth address in the data to sign), and charlie tries to send the tx from hsi own account.

Reply:
Ah sorry, yes.